Scripts/globals.gd

```diff
@@ -0,0 +1,29 @@
+extends Node
+
+signal speed_updated
+
+var boat_speed = 0
+var boat: Boat
+
+enum Task_type {
+	ROWING,
```

Comment:
For those looking at this and seeing the trailing comma, Godot actually recommends this for easier refactoring/updating: https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html#formatting